portafly/src/i18n/locales/en/applications.yml

```diff
@@ -0,0 +1,7 @@
+---
+page_title: Applications | Portafly
```

Comment:
@porueesq @dfennessy 
This is the format we are going to use for the strings.
Please tell us if you see anything wrong
Note: See all .yml files

Reply:
@damianpm Can you please show us an example using variables in the text? Examples:
_Hello **John**, you have **5** requests to approve_ (username, plural noun)
_Hello **John**, you have **1** requests to approve_ (username, singular noun)

Reply:
@porueesq 
```
example_interpolation: "Hello <strong>{{name}}</strong>, you have {{count}} request to approve"
example_interpolation_plural: "Hello <strong>{{name}}</strong>, you have {{count}} requests to approve"
```
The plural form requires the suffix `_plural`

If `name = 'John'` and `count = 1`, it will render: Hello **John**, you have 1 request to approve

If `name = 'John'` and `count = 20`, it will render: Hello **John** you have 20 requests to approve

Reply:
I've added this to our [Microcopy Guide](https://docs.google.com/document/d/1LBkuJdItYpdwa3-ewLp7gqGsj0VJxDWhp4w1ijKBqdc/edit#heading=h.i12tgogjrdx6), under _3scale-specific guidelines_